src/Microsoft.AspNet.Mvc.Razor/DefaultViewLocationCache.cs

```diff
@@ -40,9 +45,14 @@ public ViewLocationCacheResult Get([NotNull] ViewLocationExpanderContext context
 
         /// <inheritdoc />
         public void Set(
-            [NotNull] ViewLocationExpanderContext context,
-            [NotNull] ViewLocationCacheResult value)
+            ViewLocationExpanderContext context,
+            ViewLocationCacheResult value)
```

Comment:
Nice. Does your module detect value types and ignore `[NotNull]` when removing it?